### ISSUE TITLE
IDE changed

### ISSUE THREAD

**Issue by Erico103** [stackblitz/core#1947]
Hello, The IDE of my workspace  changed and I do not know how to fix it. In the past I could change the projectname I am working on in the top bar but I cannot longer do that, either can I change the projectname somewhere else. I wonder if this is a bug or something I could fix somewhere?

**Comment by donmckenna** [comment]
Do you see the "Edit project info" dropdown?

![image](https://user-images.githubusercontent.com/22125230/182659243-5c38526c-3148-435e-b489-889dd873a338.png)

![image](https://user-images.githubusercontent.com/22125230/182659178-010ac2be-ba28-4a11-8ab6-61ea406ab85a.png)

**Comment by Erico103** [comment]
Hi donMckenna ty for replying. But no I do not see that on the top bar or anywhere else in the IDE. This attached file shows you how I see it.
![picture](https://user-images.githubusercontent.com/85289051/182818477-86ab7ecd-bfb5-454f-a364-f00ee0b6bd90.png)
.

**Comment by donmckenna** [comment]
I see you closed this but for an explanation, in the view you showed the mobile styles had kicked in, which does restrict the elements in the top bar, including the "Edit project info" elements.

If you can increase your browser size, or view the project on a larger screen, you should see them again.

Though we understand this isn't ideal and are working on a better solution for mobile.
Thanks for your input!

**Comment by Erico103** [comment]
Hi @donmckenna , thank you for your help. I kinda feel silly now.  I forgot that I enlarged the view with ctrl scrollbutton(I did this cause the fonts were too small and had problems reading it), apparantly I accidentally scrolled again and made these elements dissapear. I closed this cause I put the browser back to 100 procent and it was back again. I allready found out that I can change the font size in the usersettings which has effect on the code area of the IDE. The rest is still small though like the notation of the pages and the edit project info. But that is not a big problem, if I can read the code clearly is what matters. Thank you again for the help!
